include/mbgl/platform/default/glfw_view.hpp

```diff
@@ -25,10 +25,16 @@ class GLFWView : public mbgl::View {
     static void onMouseClick(GLFWwindow *window, int button, int action, int modifiers);
     static void onMouseMove(GLFWwindow *window, double x, double y);
 
+    // Callback called when the user presses the key mapped to style change.
+    // The expected action is to set a new style, different to the current one.
+    void setChangeStyleCallback(std::function<void()> callback);
+
     void run();
     void fps();
 
-public:
+    GLFWwindow *window = nullptr;
```

Comment:
> Why is this public?

Everything was public before. This one was kept because of:
`linux/main.cpp:31:        glfwSetWindowShouldClose(view->window, true);`